### PR TITLE
Removes ling from snaxi.

### PR DESCRIPTION
**Pull request by Allspice**
0% tested. I'm not sure I've ever played in a snaxi ling round, but it doesn't sound good, and I've heard some bitching about it every now and then.

:cl:
* rscdel: Remove ling from snaxi.

### CODE REVIEW

**Comment by BomberBro** [comment]
While normally I'd say yeah it sucks, if you're gonna make a PR about this you really should play a round with it and see for yourself whether you think it should get shitcanned.

**Comment by DamianX** [comment]
noooo you cant remove features we havent added any antag since antag datums!!

**Comment by d3athrow** [comment]
@DamianX where are the antags you added? or did you just make dumber and dumber commits in an arms race to see how you could get decollab'd?

**Comment by DamianX** [comment]
I don't like your tone, chicken. I'm still waiting for your apology.

**Comment by Baronzegit** [comment]
That was me, i did the bitching.
Although i haven't seen a good old ling-shuffle round in a while due to snaxi not being played a lot, but i still hate the ling with a passion.

Ling is the jobby silent shuffler of the antag hierarchy. People will say that you "don't have to silent shuffle" as ling but it always boils down to lings going around, picking off single targets silently until they get to the 15 genes mark at which point they use the horror form and get robusted in like 5 mins after crew corners them. Vampire, even through its faults, is a superior antag to ling despite being very similar in a lot of ways simply because you have the option to go loud extremely early on.

Frankly fuck ling. It is a shitty antag that needs to die and get replaced by "The Thing" antag from the Carpenter movie.
Thank you for reading my blog post, subscrool to my wordpress page.

**Comment by ShiftyRail** [comment]
0. Don't be dumb
1. We do need new antags, but especially fun and well-designed antags
2. Ling and Snaxi might mesh well thematically, but the bad parts of Ling and the bad parts of Snaxi coalesce into something terrible

File: maps/snaxi.dm [hunk 1]
```diff
@@ -159,7 +159,7 @@
 	if(ispath(DR.role_category,/datum/role/blob_overmind))
 		return FALSE
 	if(ispath(DR.role_category,/datum/role/changeling))
```
Comment (boy2mantwicethefam) [inline maps/snaxi.dm:161]: You can remove this entire ling check

**Comment by 9percents** [comment]
about time

**Pull request merged**